pkg/datapath/linux/node.go

```diff
@@ -426,6 +426,14 @@ func (n *linuxNodeHandler) NodeUpdate(oldNode, newNode node.Node) error {
 }
 
 func (n *linuxNodeHandler) enableIPsec(newNode *node.Node) {
+	upsertIPsecLog := func(err error, spec string, loc, rem *net.IPNet) {
+		if err != nil {
+			log.WithError(err).Errorf("Upsert %s local %s remote %s", spec, loc, rem)
```

Comment:
Usually we include additional values through `log.WithFields()`, which allows log scrapers to easily find the terms:

```
scopedLog := log.WithFields(logrus.fields{
    logfields.Reason: spec,
    "local-ip": loc,
    "remote-ip": rem,
});
if err != nil {
    scopedLog.WithError(err).Error("IPSec enable failed")
} else {
    scopedLog.Debug("Enabled IPsec")
}
```

Reply:
Yeah I sort of reinvented a poor version of WithFIelds will use standard approach.